### PR TITLE
Registration flow changed

### PR DESCRIPTION
**Pull request by muak50**
Organization flow changed

### CODE REVIEW

File: lms/djangoapps/onboarding/forms.py [hunk 1]
```diff
@@ -194,17 +242,60 @@ def clean_language(self):
...
         return self.cleaned_data['start_month_year']
 
+    def clean_hours_per_week(self):
+        if self.instance.organization:
+            if not self.cleaned_data['hours_per_week']:
+                raise forms.ValidationError(
+                    ugettext_noop(EMPTY_FIELD_ERROR.format('Hours per week')))
+        return self.cleaned_data['hours_per_week']
+
+        return organization_name
```
Comment (mabdullahz) [inline lms/djangoapps/onboarding/forms.py]: Are both of these return statements properly-indented and reachable?

File: lms/djangoapps/onboarding/forms.py [hunk 2]
```diff
@@ -111,44 +117,86 @@ class UserInfoModelForm(BaseOnboardingModelForm):
...
+        }
 
         self.fields['country_of_employment'].required = False
         self.fields['city_of_employment'].required = False
 
-        if not self.instance.organization:
-            self.fields['role_in_org'].required = False
-            self.fields['start_month_year'].required = False
-            self.fields['hours_per_week'].required = False
+        # if not self.instance.organization:
```
Comment (ahmed-faraz46) [inline lms/djangoapps/onboarding/forms.py]: Let's please verify this commented code

File: lms/djangoapps/onboarding/forms.py [hunk 3]
```diff
@@ -262,9 +358,81 @@ def save(self, request, commit=True):
...
+        first_name = user.first_name
+        last_name = user.last_name
+
+        if not is_currently_unemployed:
+            organization_name = self.cleaned_data.get('organization_name', '').strip()
+            is_poc = self.cleaned_data['is_poc']
+            org_admin_email = self.cleaned_data['org_admin_email']
+
+        if not is_currently_unemployed and organization_name:
+            organization_to_assign = Organization.objects.filter(label__iexact=organization_name).first()
```
Comment (ahmed-faraz46) [inline lms/djangoapps/onboarding/forms.py]: Let's please use get or create.

File: lms/djangoapps/onboarding/forms.py (hunk 1 above)
Comment (muak50) [reply lms/djangoapps/onboarding/forms.py]: Thanks, Done

File: lms/djangoapps/onboarding/forms.py (hunk 3 above)
Comment (muak50) [reply lms/djangoapps/onboarding/forms.py]: It's old code... so just leaving it for now.

**Pull request merged**